src/zcl/definition/cluster.ts

```diff
@@ -4106,6 +4106,51 @@ const Cluster: {
         commands: {},
         commandsResponse: {}
     },
+    liXeePrivate: {
+        ID: 0xFF66,
```

Comment:
Please add a manufacturer code here (like https://github.com/Koenkk/zigbee-herdsman/blob/d50c3b101757cc27f19fcd4afecfb672880cc04a/src/zcl/definition/cluster.ts#L4210)

Reply:
Just as note for people interested in finding the code: https://github.com/fairecasoimeme/Zlinky_TIC/blob/7f10743f5cb6c3c1bde4d5aaee412b99e45bfdbf/ZLinky/Build/mcux/Makefile#L146

Or get a OTA image, go to offeset 10 and look the next 2 bytes (LE encoding)